src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -536,10 +536,11 @@ private static VersionInfo GetVersionInfo()
         public override ISchema Schema => _bindings;
 
         /// <summary>
-        /// Convenience constructor for public facing API.
+        /// Convenience constructor for creating <see cref="ConcatTransform"/>.
```

Comment:
>Convenience constructor for creating <see cref="ConcatTransform"/>. [](start = 12, length = 67)

Repetitive of the line below it, IMO.

Reply:
My current description is based on "what it is, what it does and the little description of the  class".

---
In reply to: [207687092](https://github.com/dotnet/machinelearning/pull/646#discussion_r207687092) [](ancestors = 207687092)

Reply:
i think the "Convenience" word is unnecessary. Also, constructor implies it constructs. 

Maybe just "Initializes <see cref="ConcatTransform"/>"

---
In reply to: [207976998](https://github.com/dotnet/machinelearning/pull/646#discussion_r207976998) [](ancestors = 207976998,207687092)